tools/proteomiqon_labeledproteinquantification/labeledproteinquantification.xml

```diff
@@ -0,0 +1,2527 @@
+                <option value="All" selected="true">All</option>
+                <option value="No">No</option>
+                <option value="UseOnly">Use Only</option>
+            </param> 
+            <when value="All">
+            </when>
+            <when value="No">
+            </when>
+            <when value="UseOnly">
+                <param name="ModificationFilterSequence" type="select" label="Sequence" multiple="true">
+                    <option value="Acetylation'ProtNTerm'">Acetylation'ProtNTerm'</option>
+                    <option value="Carbamidomethyl'Cys'">Carbamidomethyl'Cys'</option>
+                    <option value="Oxidation'Met'">Oxidation'Met'</option>
+                    <option value="Phosphorylation'Ser'Thr'Tyr'">Phosphorylation'Ser'Thr'Tyr'</option>
+                    <option value="Pyro_Glu'GluNterm'">Pyro_Glu'GluNterm'</option>
+                    <option value="Pyro_Glu'GlnNterm'">Pyro_Glu'GlnNterm'</option>
+                </param>
+            </when>
+        </conditional>
+        <section name="GlobalAggregationParams" title="Aggregate Global Modifications Parameters">
+            <expand macro="AggregationParams" />
+        </section>
+        <section name="AggregatePeptideChargeStatesParams" title="Aggregate Peptide Charge States Parameters">
+            <conditional name="AggregatePeptideChargeStatesParamsCond">
+                <param name="AggregatePeptideChargeStatesParamsEnabled" type="select" label="Specify Aggregate Peptide Charge States Parameters?">
+                    <option value="yes">Yes</option>
+                    <option value="no" selected="true">No</option>
+                </param> 
+                <when value="yes">
+                    <expand macro="AggregationParams" />
+                </when>
+                <when value="no">
+                </when>
+            </conditional>
+        </section>
+        <section name="AggregateModifiedPeptidesParams" title="Aggregate Modified Peptides Parameters">
+            <conditional name="AggregateModifiedPeptidesParamsCond">
+                <param name="AggregateModifiedPeptidesParamsEnabled" type="select" label="Specify Aggregate Modified Peptides Parameters?">
+                    <option value="yes">Yes</option>
+                    <option value="no" selected="true">No</option>
+                </param> 
+                <when value="yes">
+                    <expand macro="AggregationParams" />
+                </when>
+                <when value="no">
+                </when>        
+            </conditional>
+        </section>
+        <section name="AggregateToProteinGroupsParams" title="Aggregate To Protein Groups Parameters">
+            <expand macro="AggregationParams" />
+        </section>
+        <param name="outputParamfile" type="boolean" value="false" label="Output parameter file"/>
+    </inputs>
+    <outputs>
+        <data format="txt" name="out_labeledQuant" />
```

Comment:
Is that a tabular?

Reply:
Well, the tool produces a .txt but I suppose the content is tabular anyways, so I will change it to tabular!

Reply:
If its tab separated its a tabular :)